kotest-assertions/kotest-assertions-json/src/commonMain/kotlin/io/kotest/assertions/json/compare.kt

```diff
@@ -20,8 +18,8 @@ enum class CompareMode {
    /**
     * Compare by value, coercing if possible.
     *
-    * For example, "true" and true will match because the string value can be coerced into a valid boolean.
-    * Similarly, "100" and 100 will match as the former can be coerced into an int.
+    * For example, `true` and `true` will match because the string value can be coerced into a valid boolean.
```

Comment:
I do not understand the change from 
```
"true" and true will match
```
to
```
`true` and `true` will match
```
after the change, both tokens are identical. Same goes for the change to
```
"true" and true will match
```
What is the difference between these two tokens?

Reply:
Oops, that's a mistake. IntelliJ was too helpful.

```suggestion
    * For example, `"true"` and `true` will match because the string value can be coerced into a valid boolean.
```

Reply:
ah, the PR is closed, I'll make a new one.

Reply:
https://github.com/kotest/kotest/pull/4228